### PR TITLE
 Switch from JSHint to ESLint, fixes #20 

### LINKED ISSUE #20: Switch from JSHint to ESLint
ESLint's ES6, JSX, and plugin support are going to be essential for this repository.

### PR DESCRIPTION
**Pull request by nkbt**
`.eslintrc` is taken from `prod-module-boilerplate`

### CODE REVIEW

File: app/test/unit/server/index.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ module.exports = function client() {
         var
           body = res.text,
 
-          contains = function (body) {
-            return JSON.stringify(body).indexOf('Hello, world!') !== -1;
+          contains = function (text) {
```
Comment (nkbt) [inline app/test/unit/server/index.js]: `body` was defined in upper scope, so should have been renamed.

File: package.json [hunk 2]
```diff
@@ -34,17 +34,18 @@
     "version-healthcheck": "^0.1.2"
   },
   "devDependencies": {
-    "minifyify": "^7.0.1",
     "array-intersection": "^0.1.2",
+    "babel-eslint": "^3.1.17",
+    "browserify": "^10.2.4",
```
Comment (nkbt) [inline package.json]: npm sorted list of deps

File: app/test/unit/server/index.js (hunk 1 above)
Comment (ericelliott) [inline app/test/unit/server/index.js]: Rename to `response` or `res`? It's not actually text until it gets stringified. =)

File: package.json (hunk 2 above)
Comment (ericelliott) [inline package.json]: :smile:

File: app/test/unit/server/index.js (hunk 1 above)
Comment (nkbt) [inline app/test/unit/server/index.js]: `res` is also defined. So `response` then...

**Comment by ericelliott** [comment]
:+1:

**Pull request merged**